source/designers-guide/snippets/index.md

```diff
@@ -59,6 +59,18 @@ While snippets are mostly used to translate plain text, they are flexible enough
 {s name="frontend/checkout/cart/separate_dispatch"}<strong>bold example text</strong>{/s}
 ```
 
+### Using dynamic snippets
+
+You can create snippets, that use dynamic snippet names. This can be of use, when you only have a value/name stored in a variable, but want a translation of this value/name.
+
+This will work with following smarty code:
+```
+{$name = "SnippetName"|cat:$sArticle.attr1}
```

Comment:
what is "SnippetName" exactly? could you write some example what value the `$name` variable now has?

and what is stored in the `attr1`, and how the snippet in the ini file must look like